src/OpenTelemetry.Instrumentation.SqlClient/README.md

```diff
@@ -60,7 +60,59 @@ For an ASP.NET application, adding instrumentation is typically done in the
 This instrumentation can be configured to change the default behavior by using
 `SqlClientInstrumentationOptions`.
 
-TODO - describe options
+### SetStoredProcedureCommandName
+
+By default, when CommandType is CommandType.StoredProcedure this
+instrumentation will set the `db.statement` attribute to the stored procedure
+command name. This behavior can be disabled by setting the
+`SetStoredProcedureCommandName` to false.
+
+The following example shows how to use `SetStoredProcedureCommandName`.
+
+```csharp
+using Sdk.CreateTracerProviderBuilder()
+    .AddSqlClientInstrumentation(
+        options => options.SetStoredProcedureCommandName = false)
+    .AddConsoleExporter()
+    .Build();
+```
+
+### SetTextCommandContent
+
+By default, when CommandType is CommandType.Text, this instrumentation will not
+set the `db.statement` attribute. This behavior can be enabled by setting
+`SetTextCommandContent` to true.
+
+The following example shows how to use `SetTextCommandContent`.
+
+```csharp
+using Sdk.CreateTracerProviderBuilder()
+    .AddSqlClientInstrumentation(
+        options => options.SetTextCommandContent = true)
+    .AddConsoleExporter()
+    .Build();
+```
+
+### EnableConnectionLevelAttributes
+
+By default, `EnabledConnectionLevelAttributes` is disabled and this
+instrumentation sets the `peer.service` attribute to the
+[`DataSource`](https://docs.microsoft.com/en-us/dotnet/api/system.data.common.dbconnection.datasource)
```

Comment:
"en-us" - please remove this from the link, so it'll auto redirect to users language.

Reply:
Nice catch! 🇺🇳